pkg/parcimonie.sh.user.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=parcimonie with config file ~/.config/parcimonie.sh.conf
+
+[Service]
+Type=simple
+EnvironmentFile=~/.config/parcimonie.sh.conf
```

Are you sure that this works?
https://unix.stackexchange.com/a/534079 suggests that systemd doesn't expand `~` characters into the user's home directory.
According to https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers perhaps it should be `%h`?

---

You are right. Thank you.

```suggestion
EnvironmentFile=-%E/parcimonie.sh.conf
```

The [`-`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=) ignores missing files without raising an error and the [`%E`](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers) will be replaced with the configuration directory root (`$XDG_CONFIG_HOME` or `$HOME/.config`).